src/resources/js/views/datepicker.js

```diff
@@ -363,61 +314,49 @@ tribe.events.views.datepicker = {};
 	 * @return {void}
 	 */
 	obj.init = function( event, index, $container, data ) {
+		$container.trigger( 'beforeDatepickerInit.tribeEvents', [ index, $container, data ] );
+
 		var $input = $container.find( obj.selectors.input );
 		var $datepickerButton = $container.find( obj.selectors.button );
 		var viewSlug = data.slug;
 		var isMonthView = 'month' === viewSlug;
-		var isWeekView = 'week' === viewSlug;
-		var minViewMode = isMonthView ? 'year' : 'month';
-		/**
-		 * @todo: use format from BE. Paul.
-		 */
-		var daysOfWeekDisabled = isWeekView ? [ 1, 2, 3, 4, 5, 6 ] : [];
+
+		// set up datepicker change event and handler
 		var changeEvent = isMonthView ? 'changeMonth' : 'changeDate';
 		var changeHandler = isMonthView ? obj.handleChangeMonth : obj.handleChangeDate;
 
-		var tribeL10nDatatables = window.tribe_l10n_datatables || {};
-		var datepickerI18n = tribeL10nDatatables.datepicker || {};
-		var nextText = datepickerI18n.nextText || 'Next';
-		var prevText = datepickerI18n.prevText || 'Prev';
-
+		// set up datepicker button state
 		var state = {
 			isTarget: false,
 		};
 
-		var observer = new MutationObserver( obj.handleMutation( { container: $container } ) );
+		// set up mutation observer
+		obj.observer = new MutationObserver( obj.handleMutation( { container: $container } ) );
+
+		// set options for datepicker
+		obj.options.container = $container.find( obj.selectors.datepickerContainer );
+		obj.options.minViewMode = isMonthView ? 'year' : 'month';
+		var tribeL10nDatatables = window.tribe_l10n_datatables || {};
+		var datepickerI18n = tribeL10nDatatables.datepicker || {};
+		var nextText = datepickerI18n.nextText || 'Next';
+		var prevText = datepickerI18n.prevText || 'Prev';
```

Comment:
No need for this extra string here. It should always come from the localize var.

Reply:
You're right, we should always have this available, but the 2 lines above I've set empty objects if in the off case `tribe_l10n_datatables` does not exist. So `nextText` and `prevText` from `datepickerI18n` object will be undefined. `Next` and `Prev` are fallbacks.